contracts/drafts/issuance/IssuanceAdvanced.sol

```diff
@@ -27,8 +31,8 @@ contract IssuanceAdvanced is Ownable, StateMachine, ReentrancyGuard {
     event InvestmentAdded(address investor, uint256 amount);
     event InvestmentCancelled(address investor, uint256 amount);
 
-    IERC20 public currencyToken;
-    IERC20Mintable public issuanceToken;
+    ERC20MintableDetailed public currencyToken;
+    ERC20MintableDetailed public issuanceToken;
```

Comment:
1 /
Before approving this, I would like you to try to use `address` as the type variable for these two and write in a comment, what is the price difference when deploying. Plus, it will make a difference when reading as well, I believe. I would like to compare the difference.

Reply:
Ok, so the tests below are for `IssuanceAdvanced.sol`:
a. mean average gas used for deploying, with contract var storage, without instantiation when used: 2555300
b. mean average gas used for deploying, with address var storage, with contract instantiation when used: 2547700